File: week_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int stairHeigth;
    do
    {
        stairHeigth = get_int("Height: ");
    }
    while (stairHeigth > 8 || stairHeigth <= 0);

    for (int i = 1; i <= stairHeigth; i++)
    {
        // prints first columns spaces
        for (int k = stairHeigth - i; k > 0; k--)
        {
            printf(" ");
        }
        // prints first columns #
        for (int l = 0; l < i; l++)
        {
            printf("#");
        }
        // prints middle spaces
        printf("  ");

        // prints second columns
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: week_6/sentimental-mario-more/mario.py
import cs50

while True:
    height = cs50.get_int("Height: ")
    if height > 0 and height < 9:
        break
for i in range(1, height+1, 1):

    # prints first columns spaces
    for k in range(height-i):
        print(" ", end="")

    # prints first columns #
    for l in range(i):
        print("#", end="")

    # prints middle spaces
    print("  ", end="")

    # prints second columns
    for j in range(i):
        print("#", end="")
    print()
